install.html

```diff
@@ -7,9 +7,85 @@
 <script src=docs.js></script>
 
 <h1>Selenium Installation</h1>
+Selenium set up is quite different than other commercial tools. You don't install a tool using
+selenium binaries but set up your test automation project to use selenium binary in the langauge
+of your choice. The selenium binary you would need depends on browsers you want to run
+your tests upon.
 
 
+<h2>Using browser specific jar</h2>
+If you want to run tests in only a spcific brower then you can add browser specific dependency in
+your pom.xml file. You should add following dependency in your pom.xml file to run
+tests only on Firefox browser -
+
+
+<textarea rows="6" cols="100" style="border:none; margin-top:10px; margin-bottom:10px;">
```

Comment:
I see your point. 

I had raised concerns about writing code examples in html and dealing with encoding (for <, > etc) in past. It was far easier to write existing [selenium doc](www.seleniumhq.org/docs/) in rst.
I wonder how encouraging would it be for others/future doc authors to deal with encoding when writing code examples

I am curious to know what other doc authors think of it.